rotkehlchen/db/snapshots.py

```diff
@@ -54,9 +54,9 @@ def get_timed_balances(
                     DBAssetBalance(
                         category=BalanceType.deserialize_from_db(data[0]),
                         time=data[1],
-                        amount=data[2],
+                        amount=FVal(data[2]),
```

Comment:
It would make sense to create a `DBAssetBalance.deserialize_from_db()` function and use it here